apps/widget-loader/README.md

```diff
@@ -169,3 +169,56 @@ const App = ({}) => {
 
 export default App
 ```
+
+## Adding an App within Another App using Widget Loader
+
```

Comment:
Could you please mention here that this case is particularly useful in Dev mode?